client/blocks/upload-image/index.jsx

```diff
@@ -246,6 +253,19 @@ class UploadImage extends Component {
 		this.setState( { uploadedImage: null } );
 	};
 
+	componentWillMount() {
+		// use defaultImage as uploadedImage if set.
+		const { defaultImage } = this.props;
+		if ( defaultImage ) {
+			this.setState( {
+				uploadedImage: defaultImage,
+				selectedImage: defaultImage.URL,
```

Comment:
`defaultImage.URL` points to the original file, which can be many MBs big. Maybe one of the smaller thumbnail versions (URLs can be constructed with `MediaUtils.url` would be sufficient? How does that interact with editing? Do we need the original image to do editing on it? I don't know, but it's definitely worth exploring.